image_width: 640
image_height: 480
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [205.70635,   0.     , 319.5374 ,
           0.     , 205.70239, 239.52172,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.000079, -0.000039, 0.000002, 0.000033, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [205.37749,   0.     , 319.08728,   0.     ,
           0.     , 205.28555, 239.02485,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
